Examples/GRDBDemo/GRDB/GRDBQueryKey.swift

```diff
@@ -3,6 +3,39 @@ import GRDB
 import Sharing
 import SwiftUI
 
+extension DependencyValues {
+  public var defaultDatabase: DatabaseQueue {
+    get { self[GRDBDatabaseKey.self] }
+    set { self[GRDBDatabaseKey.self] = newValue }
+  }
+
+  private enum GRDBDatabaseKey: DependencyKey {
+    static var liveValue: DatabaseQueue {
+      reportIssue("""
+        A blank, in-memory database is being used for the app. To set the database that is used by \
+        the 'grdbQuery' key you can use the 'prepareDependencies' tool as soon as your app \ 
+        launches, such as in the entry point:
+        
+        @main
+        struct EntryPoint: App {
+          init() {
+            prepareDependencies {
+              $0.defaultDatabase = DatabaseQueue(…)
+            }
+          }
+        
+          // ...
+        }
```

Comment:
Let's indent this to set it apart:

```suggestion
            @main
            struct EntryPoint: App {
              init() {
                prepareDependencies {
                  $0.defaultDatabase = DatabaseQueue(…)
                }
              }

              // ...
            }
```